app/views/help/index.html.erb

```diff
@@ -8,50 +8,32 @@
-        <li>
-          <a href="/help/about-govuk" class="govuk-link govuk-!-font-size-19 govuk-!-font-weight-bold"><%= t('help.index.about') %></a>
-          <p class="govuk-body govuk-!-margin-top-1"><%= t('help.index.about_description') %></p>
-        </li>
-        <li>
-          <a href="/help/accessibility" class="govuk-link govuk-!-font-size-19 govuk-!-font-weight-bold"><%= t('help.index.accessibility') %></a>
-          <p class="govuk-body govuk-!-margin-top-1"><%= t('help.index.accessibility_description') %></p>
-        </li>
-        <li>
-          <a href="/help/beta" class="govuk-link govuk-!-font-size-19 govuk-!-font-weight-bold"><%= t('help.index.beta') %></a>
-          <p class="govuk-body govuk-!-margin-top-1"><%= t('help.index.beta_description') %></p>
-        </li>
-        <li>
-          <a href="/help/browsers" class="govuk-link govuk-!-font-size-19 govuk-!-font-weight-bold"><%= t('help.index.browsers') %></a>
-          <p class="govuk-body govuk-!-margin-top-1"><%= t('help.index.browsers_description') %></p>
-        </li>
-        <li>
-          <a href="/help/cookies" class="govuk-link govuk-!-font-size-19 govuk-!-font-weight-bold"><%= t('help.index.cookies') %></a>
-          <p class="govuk-body govuk-!-margin-top-1"><%= t('help.index.cookies_description') %></p>
-        </li>
-        <li>
-          <a href="/help/update-email-notifications" class="govuk-link govuk-!-font-size-19 govuk-!-font-weight-bold"><%= t('help.index.email') %></a>
-          <p class="govuk-body govuk-!-margin-top-1"><%= t('help.index.email_description') %></p>
-        </li>
-        <li>
-          <a href="/help/privacy-notice" class="govuk-link govuk-!-font-size-19 govuk-!-font-weight-bold"><%= t('help.index.privacy_notice') %></a>
-          <p class="govuk-body govuk-!-margin-top-1"><%= t('help.index.privacy_notice_description') %></p>
-        </li>
-        <li>
-          <a href="/help/report-vulnerability" class="govuk-link govuk-!-font-size-19 govuk-!-font-weight-bold"><%= t('help.index.vulnerability') %></a>
-          <p class="govuk-body govuk-!-margin-top-1"><%= t('help.index.vulnerability_description') %></p>
-        </li>
-        <li>
-          <a href="/help/reuse-govuk-content" class="govuk-link govuk-!-font-size-19 govuk-!-font-weight-bold"><%= t('help.index.reuse_content') %></a>
-          <p class="govuk-body govuk-!-margin-top-1"><%= t('help.index.reuse_content_description') %></p>
-        </li>
-        <li>
-          <a href="/help/terms-conditions" class="govuk-link govuk-!-font-size-19 govuk-!-font-weight-bold"><%= t('help.index.terms') %></a>
-          <p class="govuk-body govuk-!-margin-top-1"><%= t('help.index.terms_description') %></p>
-        </li>
-      </ul>
+      <%= render "govuk_publishing_components/components/list", {
+        items: [
+          sanitize("<a href='/help/about-govuk' class='govuk-link govuk-!-font-size-19 govuk-!-font-weight-bold'>" + t('help.index.about') + "</a>
```

Comment:
Non-blocking: the `govuk-!-font-size-19` class is no longer required as the default font-size from govuk-frontend is now 19px, this could be removed from each anchor element

Reply:
A really nice spot, fixed 👍